### PR TITLE
add rdf:dirLangString, rdf:reifies, rdf:Proposition, and modify rdf:JSON

### PR DESCRIPTION
**Pull request by domel**
This PR updates http://www.w3.org/1999/02/22-rdf-syntax-ns#. It modifies the Turtle file and changes from v1.1 to v1.2.

Added 

- `rdf:dirLangString`
- `rdf:reifies`, 
- `rdf:Proposition`

Modified 

- `rdf:JSON`

### CODE REVIEW

**Review by gkellogg — APPROVED** [review]

**Review by niklasl — COMMENTED** [review]
This looks good.

I presume we're making this repository the official source of this vocabulary? If this is correct I gladly welcome it.

I just have one wish for that: to first commit the original (the turtle as served on the official IRI), and then commit these changes upon that. To make it clear exactly what we're adding/changing.  (I.e. do a rebase so this change is shown as a distinct commit (or commits).)

(Also adding the `rdfs:` source as `rdfs.ttl` seems appropriate?)

**Comment by gkellogg** [comment]
I agree with @niklasl that we should start with a base which is based on the RDF 1.1 version of the RDF Vocabulary (and the RDFS Vocabulary, as well). We might also consider providing the namespace documents in different formats: The [JSON-LD Namespace files](https://github.com/w3c/json-ld-wg/tree/main/ns) generate Turtle, JSON-LD, and HTML versions of the JSON-LD vocabulary, and we could similarly do that for the RDF(S) vocabularies.

I would put `rdf.ttl` in an "ns" sub-directory, along with `rdfs.ttl`.

**Comment by niklasl** [comment]
Can this be on hold until #33 is merged, and then rebased upon that?

**Comment by domel** [comment]
@niklasl I closed this one temporarily. When `ns` directory will be merged I will create a new PR only with `rdf:dirLangString`, `rdf:reifies`, `rdf:Proposition`, `rdf:JSON`

**Pull request closed without merging**
